appveyor.yml

```diff
@@ -5,12 +5,19 @@ branches:
   only:
     - l10n_master # The branch that Crowdin targets when updates are made on .po files
     - master      # The branch that is published on Nuget.org
+    - l10n-pack   # For testing purpose
+environment:
+  packageVersion: 1.0.0
 init:
   - git config --global core.autocrlf true
+  - ps: $env:buildVersion = "$env:packageVersion.$env:appveyor_build_number"
+  - ps: $env:nugetVersion = "$env:packageVersion-beta-$env:appveyor_build_number"
```

Comment:
Can we move the version prefix to its custom variable? (beta)

Reply:
We can move the `-beta-$env:appveyor_build_number` to something like $prelease or $qualifier, but I don't think this will add a value to the `pack.ps`

Reply:
It has as much value as the variable `packageVersion: 1.0.0` that you created. And I don't mind either merging these two in a single value.

Reply:
As I said before if we will no longer use preleases or qualifier in versioning it's fine to use only one variable